utils/src/command.rs

```diff
@@ -68,9 +69,17 @@ impl CommandRunExt for Command {
     fn run(&mut self) -> Result<()> {
         let stderr = tempfile::tempfile()?;
         self.stderr(stderr.try_clone()?);
+        tracing::trace!("exec: {self:?}");
```

Comment:
I kinda hate that we'll have:

```
DEBUG exec: "lsblk" "--pairs" "--paths" "--inverse" "--output" "NAME,TYPE" "composefs"
TRACE exec: "lsblk" "--pairs" "--paths" "--inverse" "--output" "NAME,TYPE" "composefs"
```

But we can't append state to `Command` so not sure how to avoid it

Reply:
Fixed that